deeprank/utils/cal_hitrate_successrate.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
```

Comment:
Extract the calculation of hit/success rate from plot_utils to this new file, which could avoid CI failure due to the import of `rpy2` module.